CONTRIBUTING.md

```diff
@@ -74,8 +75,23 @@ Build the SDK:
 npm run build
 ```
 
+*Build issues?* See [BUILD-ISSUES.md](./BUILD-ISSUES.md) for help.
+
 ### Running Tests
 
+#### SauceLabs
+
+The SDK uses [SauceLabs](https://saucelabs.com/) to run its tests. Sign in to retrieve your *USERNAME* and *ACCESS KEY* from [User Settings](https://saucelabs.com/beta/user-settings) and add them to your `.env` file: 
+
```

Comment:
Is sauce labs actually required? My understanding is for most people they don't need sauce credentials and can run tests locally. Sauce is only used for CI.

Reply:
When I tried to run the basic (complete) suite of tests it failed because it needed the username and access key (and tunnel ID, ultimately) from Sauce. I also picked a couple packages at random to scope it and it failed as well. Maybe not all packages need it, but getting the basics up an running requires it. 

It looks like there are 61 packages. I could do some digging and see what requires what.

Reply:
You should be able to run all the tests locally without sauce from my understanding as well. Curious to see your findings.